src/test_lib_json/main.cpp

```diff
@@ -2268,6 +2267,7 @@ JSONTEST_FIXTURE_LOCAL(StyledStreamWriterTest, multiLineArray) {
     JSONTEST_ASSERT_STRING_EQUAL(expected, result);
   }
   {
+    Json::StyledStreamWriter writer;
```

Comment:
I don't really understand how *adding* a `writer` would remove a shadowed variable. I'm sure it's fine, but could you explain it?

Reply:
In the original code the line  2237 shadows line 2261 below it, but is not used unit line 2270.

Scope the variable called "writer" in the blocks that need them.  It may also have been OK to remove line 2261 in the original code, but I was not sure what the intent of making a new write was.

Reply:
I forgot to remove line 2261 when i wrote this testcase.